frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-transport-patient-count-request/radiogram-card-content-transport-patient-count-request.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="radiogram$ | async as radiogram">
+    <h5 class="card-title">Anfrage nach Patientenzahlen</h5>
+    <p *ngIf="region$ | async as region">
+        Die Transportorganisation in dieser Region möchte die Patientenzahlen in
+        den von ihr verwalteten Regionen erfahren.
+    </p>
+</ng-container>
```

Comment:
As far as I can see, you don't need the radiogram or the region at all. Therefore, you might as well remove them from the template and from the component